test/plotting.jl

```diff
@@ -0,0 +1,30 @@
+@testset "ZonedDateTime plot recipe" begin
+    start_zdt = ZonedDateTime(2017,1,1,0,0,0, tz"EST")
+    end_zdt = ZonedDateTime(2017,1,1,10,30,0, tz"EST")
+    zoned_dates = start_zdt:Hour(1):end_zdt
+
+    expected_dates = DateTime.(zoned_dates, Local)
+
+    @testset "No label" begin
+        result, = RecipesBase.apply_recipe(Dict{Symbol, Any}(), zoned_dates, 1:11)
+        xs, ys = result.args
+        @test xs == expected_dates
+        @test ys == 1:11
+        @test result.plotattributes[:xguide] == "(timezone: EST)"
+    end
+
+    @testset "label (should append to it)" begin
+        result, = RecipesBase.apply_recipe(Dict{Symbol, Any}(:xguide=>"Hi"), zoned_dates, 1:11)
```

Comment:
How about being a little more informative than `"Hi"`?
```suggestion
        result, = RecipesBase.apply_recipe(Dict{Symbol, Any}(:xguide=>"X-Axis"), zoned_dates, 1:11)
```